src/releng_docs/shipit_signoffs

```diff
@@ -0,0 +1 @@
+../shipit_signoff/docs
```

Comment:
I think we have 2 options :

(1) we can spend some time fixing
(2) we can also leave as it is until we figure out how to solve (1) at a later point

Right now I think the most value is that we merge this PR as soon as possible and I'm more in favor of option (2) as we will move ``shipit_*`` stuff out of this repo anyway in near future. At this point we will have to think about fixing things as livereload, etc...

Apart from that, can you briefly (if possible) show me how you use ``git log`` and how the above "reverse"-symlink breaks your workflow. If it is to complicated to write it don't bother, maybe you can show me before/after some meeting. I just want to understand your workflow and maybe use it myself :)